lib/components/term.js

```diff
@@ -169,6 +170,14 @@ export default class Term extends React.PureComponent {
 
     if (props.onTitle) {
       this.disposableListeners.push(this.term.onTitleChange(props.onTitle));
```

Comment:
Is this still needed?

Reply:
this line changes the tab title, but the PR wants to change the page title to identify different hyper windows.
https://github.com/zeit/hyper/blob/621b509a0685323a8a55e04850dff97a79979ffa/app/index.html#L4
I think this can be achieved by a plugin, or open them on a single window, and distinguish by tab title.

Reply:
Re:

> I think this can be achieved by a plugin, or open them on a single window, and distinguish by tab title.

I have a plugin already which does what I want-
https://github.com/elementc/hyper-window-titles

I'm proposing this for inclusion in mainline hyper because it seems kind of silly to be unable to distinguish between windows of your terminal in your system tray by default.

Specific to distinguishing by tab title: I don't always have a window visible when I want to surface it. This helps in that case. The below code: 

1: Stashes the new term title on the terminal object (I think? this.term) unconditionally every time onTitleChange fires.
2: Updates the window title if the terminal that fired this is the currently active term.
3: The focus() addition below that updates the window title with the stashed term title whenever a term becomes active.

This method means that it works with tab-terms AND split-pane-terms.